### DISCUSSION TITLE
restore deleted enterprise account

### DISCUSSION THREAD

**Discussion by EternalFeather** [community/community#147194] [Enterprise] [answered]
### Select Topic Area

Product Feedback

### Body

How can I restore my deleted enterprise account or just get the history payment of it?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @EternalFeather ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
